templates/repo/create.tmpl

```diff
@@ -43,10 +43,10 @@
 						<label>{{.i18n.Tr "repo.visibility"}}</label>
 						<div class="ui checkbox">
 							{{if .IsForcedPrivate}}
-								<input name="private" type="checkbox" checked readonly>
-								<label>{{.i18n.Tr "repo.visibility_helper_forced" | Safe}}</label>
+							<input name="private" type="checkbox" checked readonly>
+							<label>{{.i18n.Tr "repo.visibility_helper_forced" | Safe}}</label>
 							{{else}}
-								<input name="private" type="checkbox" {{if .private}}checked{{end}}>
+								<input name="private" type="checkbox" {{if .private}} checked{{end}}>
```

Comment:
For the future, that change is unnecessary because there is whitespace after "checkbox"

Reply:
It's not big problem :)